### PR TITLE
make pipeline faster

### PR DESCRIPTION
**Pull request by crapStone**

### CODE REVIEW

File: .github/workflows/backend.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name: checks
```
Comment (6543) [inline .github/workflows/backend.yml:1]: if we change "test" to "checks" we have to alter protection rules of branches ...

File: .github/workflows/frontend.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      - '.github/**'
+      - '!.github/workflows/frontend.yml'
+      - 'contrib/**'
+      - 'ghs_symbols/**'
+      - 'prompts/**'
+      - 'src-tauri/**'
+      - 'templates/**'
+      - 'translations/**'
+      - '*.md'
+      - '*.yml'
```
Comment (6543) [inline .github/workflows/frontend.yml:19]:
```suggestion
    paths:
      - 'src'
      - '!src/assets/defaultStrings.json'
      - '*.json'
      - 'yarn.lock'
      - '.github/workflows/frontend.yml'
```

File: .github/workflows/backend.yml (hunk 1 above)
Comment (crapStone) [reply .github/workflows/backend.yml:1]: We don't have to change it, I tested it.

File: .github/workflows/frontend.yml (hunk 2 above)
Comment (crapStone) [reply .github/workflows/frontend.yml:19]: You forgot every config file in the root directory.

Comment (6543) [reply .github/workflows/frontend.yml:19]: `'*.json'`<- should take care of it

Comment (crapStone) [reply .github/workflows/frontend.yml:19]: But we still have to add `.js` and `.prettierignore`.

Comment (6543) [reply .github/workflows/frontend.yml:19]: ok I'll add them :)

**Review by 6543 — APPROVED** [review]

**Comment by 6543** [comment]
@crapStone you are free to trigger the final **merge**

**Comment by 6543** [comment]
@crapStone ?

**Comment by crapStone** [comment]
@6543 sorry, the mobile ui is crap and I just booted my pc

**Pull request merged**